django-stubs/db/models/fields/related.pyi

```diff
@@ -282,7 +282,7 @@ class ManyToManyField(RelatedField[Any, Any], Generic[_To, _M]):
     ) -> None: ...
     # class access
     @overload
-    def __get__(self, instance: None, owner: Any) -> ManyToManyDescriptor[_M]: ...
+    def __get__(self, instance: None, owner: Any) -> ManyToManyDescriptor[_To, _M]: ...
```

Comment:
The naming you used in ManyToManyDescriptor `_To, _Through` makes more sense to me than `_To, _M`.

Would be nice to introduce `_Through` TypeVar for `ManyToManyField` as well.